src/Symfony/Installer/Application.php

```diff
@@ -0,0 +1,66 @@
+<?php
+
+/*
+ * This file is part of the Symfony Installer package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Installer;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+use Symfony\Component\Console\Application as ConsoleApplication;
+
+/**
+ * This is main Symfony Installer console application class.
+ */
+class Application extends ConsoleApplication
+{
+    const VERSIONS_URL = 'http://get.sensiolabs.org/symfony.version';
```

Comment:
will change to `get.symfony.com` but we still don't use https here 
https://github.com/symfony/symfony-installer/blob/master/src/Symfony/Installer/SelfUpdateCommand.php#L125 ?
Is there any reason